src/server.js

```diff
@@ -65,6 +78,23 @@ const server = Hapi.server({
     router: { stripTrailingSlash: true },
     /* https://hapijs.com/api#-serveroptionsdebug */
     debug: DW_DEV_MODE ? { request: ['implementation'] } : false,
+    cache: {
+        name: 'dw_cache',
+        provider: useRedis
+            ? {
+                  constructor: require('@hapi/catbox-redis'),
+                  options: {
+                      ...config.redis,
+                      partition: 'dw_api_cache'
```

Comment:
Since we’re unlikely to reuse an external Redis server and Redis is also be used as cache by definition I suggest we change `dw_api_cache` to just `api`.

```suggestion
                      partition: 'api'
```